### PR TITLE
Add commands to build Cobalt library locally

### PR DESCRIPTION
**Pull request by jonastsai**
b/318730771

### CODE REVIEW

**Review by hlwarriner — CHANGES_REQUESTED** [review]
Thanks for adding this!

File: cobalt/site/docs/development/setup-android.md [hunk 1]
```diff
@@ -311,6 +311,22 @@ Cobalt-related output.
...
+**Partners should always use the Google prebuilt binaries from GitHub for certification
+or software release.** However, for testing or developing purpose, they can still build
+the library locally. Ex:
+
+   ```sh
+   # Create directory for evergreen-arm-softfp with build_type=qa and sb_api_version=15
+   gn gen out/evergreen-arm-softfp_qa --args="target_platform=\"evergreen-arm-softfp\" use_asan=false target_cpu=\"arm\" build_type=\"qa\" sb_api_version=15"
+
+   # Build Cobalt library
+   ninja -C out/evergreen-arm-softfp_qa
```
[the remark this answers is not in the document]
Comment (jonastsai) [reply cobalt/site/docs/development/setup-android.md:325]: Thank you Holden. How can I miss such an important parameter?!

[the remark this answers is not in the document]
Comment (jonastsai) [reply cobalt/site/docs/development/setup-android.md]: This is fixed.

File: cobalt/site/docs/development/setup-android.md [hunk 2]
```diff
@@ -311,6 +311,22 @@ Cobalt-related output.
 As with the Linux build, use the `debug`, `devel`, or `qa` configs to trace
 Cobalt's callstacks.
 
+### Build Cobalt library locally
+
+**Partners should always use the Google prebuilt binaries from GitHub for certification
+or software release.** However, for testing or developing purpose, they can still build
```
[the remark this answers is not in the document]
Comment (jonastsai) [reply cobalt/site/docs/development/setup-android.md]: This is updated.

File: cobalt/site/docs/development/setup-android.md [hunk 3]
```diff
@@ -311,6 +311,22 @@ Cobalt-related output.
...
+
+   ```sh
+   # Create directory for evergreen-arm-softfp with build_type=qa and sb_api_version=15
+   gn gen out/evergreen-arm-softfp_qa --args="target_platform=\"evergreen-arm-softfp\" use_asan=false target_cpu=\"arm\" build_type=\"qa\" sb_api_version=15"
+
+   # Build Cobalt library
+   ninja -C out/evergreen-arm-softfp_qa
+   ```
+
+   Once the Cobalt library is built, back to [Compile Android APK using Ninja](#compile-android-apk-using-ninja) to build the APK.
```
[the remark this answers is not in the document]
Comment (jonastsai) [reply cobalt/site/docs/development/setup-android.md]: This is updated.

File: cobalt/site/docs/development/setup-android.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hlwarriner) [reply cobalt/site/docs/development/setup-android.md]: Ha - you're welcome!

**Review by hlwarriner — APPROVED** [review]

File: cobalt/site/docs/development/setup-android.md [hunk 4]
```diff
@@ -311,6 +311,22 @@ Cobalt-related output.
 As with the Linux build, use the `debug`, `devel`, or `qa` configs to trace
 Cobalt's callstacks.
 
+### Build Cobalt library locally
+
+**Partners should always use the Google prebuilt binaries from GitHub for certification
```
Comment (lamdbui) [inline cobalt/site/docs/development/setup-android.md:316]: Let's directly link the GitHub binaries for quick access as well:
- [Google prebuilt binaries from GitHub](https://github.com/youtube/cobalt/releases)

**Review by lamdbui — CHANGES_REQUESTED** [review]
Thanks for improving this, Jonas! I added just one minor suggestion to link the prebuilt binaries.

**Review by yuying-y — APPROVED** [review]

File: cobalt/site/docs/development/setup-android.md (hunk 4 above)
Comment (jonastsai) [reply cobalt/site/docs/development/setup-android.md]: Thanks for the suggestion. It is updated.

**Review by lamdbui — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**